PLATFORM_README.md

```diff
@@ -127,24 +127,44 @@ For an existing Angular CLI application,
 
     For models prior to 0.11.1 use `fundamental-ngx`
 
+    ```javascript
     import { SwitchModule } from '@fundamental-ngx/core';
 
     @NgModule({
     ...
     imports: [SwitchModule],
     })
     export class DemoModule { }
+    ```
+
+3. **Add the component to your HTML.**
 
+    ```html
+    <fd-switch [size]="'l'" [(checked)]="myValue">Large Switch</fd-switch>
```

Comment:
do you want to add a platform example?

Reply:
have added a platform component example instead.